src/QueryPressure.Postgres.App/PostgresAppModule.cs

```diff
@@ -11,7 +10,7 @@ protected override void Load(ContainerBuilder builder)
     builder.RegisterType<PostgresConnectionProviderCreator>()
       .AsImplementedInterfaces();
 
-    builder.RegisterInstance(new ProviderInfo("Postgres"))
+    builder.RegisterInstance(new PostgresProviderInfo())
```

Comment:
Use just instance, not new class

Reply:
Hope that got your point properly and update registrations to use only types instead of instances

Reply:
No...
In fact it was more about using just instances without classes
new ProviderInfo(....)

But... Big string constant in DI registration is not so beautiful, so it seems to be ok as is.